update.sh

```diff
@@ -49,6 +49,8 @@ cp etc/library.sh /usr/local/etc/
 
 source /usr/local/etc/library.sh
 
+./backward_updates.sh
```

Comment:
```suggestion
./run_update_history.sh "updates"
```

What do you think about that name? also it would be nice to have the folder be a parameter

Reply:
Sure! Sounds good!